sql/2019/10_SEO/10_07b.sql

```diff
@@ -1,13 +1,21 @@
 #standardSQL
-
-# <title> length
-
+# 10_07b: <title> length
```

Comment:
I've run into a lot of sites which have more than one title tag unfortunately.

Adding this as a custom metric would be very simple, so we could update the query for October's run and see how different the data ends up being?

Reply:
This query should only measure the first title, which is what a custom metric would assumedly do too.

Implementing it as a custom metric would also cut it really close to the scheduled launch in early November, considering that we wouldn't have data until the end of October and authors/translators would need to revise the chapter contents.

Reply:
Fair enough